broker/restapisrv.go

```diff
@@ -62,35 +60,21 @@ func (apisrv *RestApiSrv) Start(cfg *Config, broker *ThinBroker) {
 		rest.Delete("/v2/subscription/#sid", apisrv.deletev2Subscription),
 
 		//NGSI-LD APIs
-
-		//create and update
+		rest.Post("/ngsi-ld/v1/notifyContext/", broker.NotifyLdContext),
 		rest.Post("/ngsi-ld/v1/entities/", broker.LDCreateEntity),
+		rest.Get("/ngsi-ld/v1/entities/#eid", apisrv.LDGetEntity),
+		rest.Get("/ngsi-ld/v1/entities", apisrv.GetQueryParamsEntities),
 		rest.Post("/ngsi-ld/v1/entities/#eid/attrs", broker.LDAppendEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs", broker.LDUpdateEntityAttributes),
 		rest.Patch("/ngsi-ld/v1/entities/#eid/attrs/#attr", broker.LDUpdateEntityByAttribute),
-
-		//query
```

Comment:
this should not be deleted, its comments which tells about the below API

Reply:
Done in other pull request

Reply:
Dene in other pull request

Reply:
Done in other pull request